pdr_backend/pdr_dashboard/util/db.py

```diff
@@ -424,11 +496,11 @@ def feeds_metrics(self) -> dict[str, Any]:
             SELECT COUNT(DISTINCT(contract, pair, timeframe, source))
             FROM {tbl_parquet_path(self.lake_dir, Prediction)}
         """
+        start_date_day = self.start_date.day if self.start_date else 0
```

Comment:
why not have this in the qurey db function itself, and pass in the cache_file_prefix instead of the full name?